index.html

```diff
@@ -3,9 +3,43 @@
   <head>
     <title>Relaxr</title>
     <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
   </head>
   <body>
+
     <header>
+    	<h1 class="head">Relaxr</h1>
+    		<h2 class ="int">Get piece of mind with a single tap</h2>
+    			<div class="button">
+    			<h3>Get it Now</h3>
+    		</div>
     </header>
+
+    <section id="benefit">
+    		<h2>Benefits</h2>
+    			<p class="ftw">The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+    <ul>
+    	<li>Schedule meetings for you</li>
+		<li>Excel automation to complete your work for you</li>
+		<li>Responds to emails on your behalf</li>
+		<li>Does all your work for you with our revolutionary AI technology</li>
+	</ul>	
+    </section>
+
+	<section id="quote">
+			<p class="ama"><q>Relaxr changed my life. I&#39ve been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work.</q></p>
```

Comment:
Nice use of the q element. I honestly did not even know that existed. A blockquote would probably serve you better though.